core/dbt/main.py

```diff
@@ -1088,6 +1088,27 @@ def parse_args(args, cls=DBTArgumentParser):
         """,
     )
 
+    schema_cache_flag = p.add_mutually_exclusive_group()
+    schema_cache_flag.add_argument(
+        "--cache_selected_only",
```

Comment:
One thing I just caught: for consistency with other flags, this should probably be `--cache-selected-only` (hyphens not underscores)